scripts/api-key.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+def update_ipynb_files(directory):
```

Comment:
I think you will also need to update all the internal links, otherwise the links in the "what's next" sections of each notebook will open the github instead of the corresponding notebook in IDX, breaking the learning path.